app/components/ui/NewsArticles/NewsArticles.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState, useEffect } from "react";
+import {
+  CarouselProvider,
+  Slider,
+  Slide,
+  ButtonBack,
+  ButtonNext,
+  DotGroup,
+} from "pure-react-carousel";
+
+import * as dataService from "../../../utils/DataService";
+
+import styles from "./NewsArticles.module.scss";
+
+interface NewsArticle {
+  id: string;
+  headline: string;
+  effective_date: string;
+  expiration_date: string;
+  body: string;
+  priority: number;
+  url?: string;
+}
+
+const formatDate = (date: string): string =>
+  new Date(date).toLocaleDateString("en-US", {
+    year: "numeric",
+    month: "short",
+    day: "numeric",
+  });
+
+const ArticleItem = ({ article }: { article: NewsArticle }) => (
+  <div className={styles.articleItem}>
+    <div className={styles.newsContent}>
+      <p className={styles.articleTitle}>
+        {article.headline} <span className={styles.dateSeperator}>|</span>{" "}
+        <span className={styles.articleDate}>
+          {formatDate(article.effective_date)}
+        </span>
+      </p>
+      <p className={styles.articleBody}>{article.body}</p>
+    </div>
+    {article.url && (
+      <a href={article.url} className={styles.articleLink}>
```

Comment:
I think we'd probably want these links to open in a new window, since they'll almost always link to an external site, right?

Reply:
Good call, updated.